### DISCUSSION TITLE
Home Down Payment

### DISCUSSION THREAD

**Discussion by Mozenraath** [ananthakumaran/paisa#132] [Q&A] [answered]
How do I account for down payment done for my apartment.
I took 50% loan and 50% from savings/checking.  Issue I faced was that under Assets category there was no. of units were doubling or the price of the unit was not correct.

![Screenshot 2024-01-08 215327](https://github.com/ananthakumaran/paisa/assets/155997609/534cd036-eeb1-4159-93e2-83e1a1599fe2)

**Comment by ananthakumaran** [comment, marked as answer]
`@` specifies the price of a **single** unit, in your case, I am assuming that is 24 Lakhs?

You can represent the purchase using a single transaction itself. There can be more than one debit/credit entries (postings). The total needs to balance to 0, there are no constrains on the number of postings.

```
2024/01/08 House
    Assets:House                                   1 APT @ 2400000 INR
    Liabilities:Loan                        -1200000 INR
    Assets:Checking                         -1200000 INR
```

**Reply by Mozenraath** [reply]
That worked ! Thanks a lot !

**Reply by Mozenraath** [reply]
Now I tried to model my actual payments with income from Euros. But Doctor is flagging following errors.

![image](https://github.com/ananthakumaran/paisa/assets/155997609/4ab373a0-16de-4f1e-9fcb-5515aec180f4)

**Reply by ananthakumaran** [reply]
I haven't tested it, but I am assuming it might be because you are adding EUR, but trying to withdraw INR directly, which might be the cause. Could you try to do `-5000 EUR @ 80 INR` when you do the home purchase. If that doesn't work, let me know, I will check.

**Reply by Mozenraath** [reply]
This way it worked. Maintaining it in same currency. any reason why ledger didn't  calculate the same with previous method?

**Reply by ananthakumaran** [reply]
In general, I think of accounts as physical jars. You put x into it and you can only take x out of it. If you try to pull y out of it (even if you have told the ledger how to convert x -> y), it will not do it, instead, your account will have negative y and x as balance. Paisa could be better here at highlighting this issue, but except the doctor page, other places will do the conversion and hide the problem from you.
